### PR TITLE
Final Group Project:- Space Travelers Hub

### PR DESCRIPTION
**Pull request by hmunish**
In this project, we performed the following tasks.

- [ ] Created a new repository and called it space-traveler's-hub.
- [ ] Set up the kanban board.
- [ ] Created React-app.
- [ ] Installed and set up the project tools and directories.
- [ ] Set up linters, testing library, redux.
- [ ] Created Navigation for different pages using routes.
- [ ] Worked on individual tasks and gave feedback to each other.
- [ ] Individually added Rockets, missions, and other functionalities.
- [ ] Updated README file.
- [ ] Performed testing of components.
- [ ] Fixed linter errors.

### CODE REVIEW

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @hmunish @cosywasswa 

Great job for implemented  the some of the Project Requirement , but there are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻
✔️ No Linter errors
✔️ Correct Git-Flow
✔️ Good commit messages
✔️ Professional README

almost

## Required Changes ♻️
_Check the comments below the review._


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/rockets/rocketsCard.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  color: #fff;
+  cursor: pointer;
+}
+
+button.rocketCard.cancelBtn {
+  background-color: #fff;
+  color: gray;
+  border: 1px solid gray;
+  box-shadow: 1px 1px 1px gray;
+}
```
[the remark this answers is not in the document]
Comment (hmunish) [reply src/components/rockets/rocketsCard.css:60]: Hi, @Abenezer-Tilahun, thanks for providing us with valuable feedback. But as me & my partner (@cosywasswa ) run the app on my system it renders the image of the rocket with proper alignment & size, as the width: 21% property has been given on the image. I have attached snapshots for your reference, kindly please review the app & assist us further. 

Thanks again for your valuable time.

rocket-2
rocket-1

[the remark this answers is not in the document]
Comment (Abenezer-Tilahun) [reply src/components/rockets/rocketsCard.css:60]: Hello @hmunish , Sorry for the confusion at all! but  It's essential to test the app on various browsers for compatibility. The appearance and layout of images can be influenced by CSS styles. It's possible that there are CSS rules specific to certain browsers, such as Firefox, which might affect how images are displayed differently compared to Chrome. As you mentioned, when I checked it on Chrome, it worked as expected. Therefore, it's crucial to review and adjust any browser-specific CSS rules to ensure consistent image rendering across different browsers. At this case next reviewer will approve the project .

[the remark this answers is not in the document]
Comment (hmunish) [reply src/components/rockets/rocketsCard.css:60]: @Abenezer-Tilahun , it's completely fine. You mean to say that I can submit it for review without amending anything the next reviewer will approve the project.

[the remark this answers is not in the document]
Comment (Abenezer-Tilahun) [reply src/components/rockets/rocketsCard.css:60]: Hello @hmunish Yes it will be approved.

**Review by cynthiainga — APPROVED** [review]
Hi @hmunish & @cosywasswa, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- [ ] Good design ✔️ 
- [ ] Users can reserve rockets and join mission ✔️ 
- [ ] No linters errors ✔️ 
- [ ] Good commit history ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**